src/index.tsx

```diff
@@ -2,13 +2,15 @@ import React from 'react';
 import ReactDOM from 'react-dom/client';
 import './index.css';
 import App from './App';
+import Login from '.../Login';
```

Comment:
This import isn't quite happy: 
<img width="443" alt="image" src="https://user-images.githubusercontent.com/126940746/224431865-eb2ebe71-78d5-42f7-a345-4e45964d72f4.png">

Reply:
<img width="1128" alt="image" src="https://user-images.githubusercontent.com/126940746/224432192-1347385f-d938-4e02-9e22-6d9fc13b938e.png">